cluster/get-kube.sh

```diff
@@ -61,16 +61,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# TODO(pipejakob): Use dl.k8s.io/bazel once the redirection uses the correct bucket.
+KUBERNETES_BAZEL_RELEASE_URL="${KUBERNETES_BAZEL_RELEASE_URL:-${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release-dev/bazel}}"
```

Comment:
It doesn't seem like this var is necessary given that we already have KUBERNTES_RELEASE_URL and kubetest can use this